compiler-cli/src/main.rs

```diff
@@ -168,6 +168,10 @@ enum Command {
         #[clap(long, ignore_case = true)]
         runtime: Option<Runtime>,
 
+        /// The module to run
+        #[clap(short, long, ignore_case = true)]
```

Comment:
Let's not ignore case, `-m MYAPP/THINGY` should probably not be accepted

Reply:
I removed `ignore_case = true`.